learnyounode/03-my-first-async-io.js

```diff
@@ -0,0 +1,9 @@
+var fs = require('fs');
+
+function callback (err, response) {
+  var array = response.toString().split('\n');
+  array.pop();
+  console.log(array.length);
+};
```

Comment:
You're not handling the error case - if `fs` hit an error, it should trigger your code to fire an error.